### ISSUE TITLE
Website typo - 'Gerring started'

### ISSUE THREAD

**Issue by adam-enko** [eclipse-mat/mat#83]
I see a typo in the website: it should say "Getting started", but it says "Gerring started".

<img width="375" alt="image" src="https://github.com/user-attachments/assets/ec89e814-c5a4-470c-8139-bb9f4907b881">

I would make a PR, but I see the typo is in two places in different repos:

1. https://github.com/eclipse-mat/website-public/blob/f6f801a16a2c3a5aa61904f5aeefff5a4efc6924/index.html#L666
2. https://github.com/eclipse-mat/website/blob/d14fbcaa28f236c25cf83eff2224b6d1ba70fb9c/config.toml#L95

**Comment by krumts** [comment]
Hi @adam-enko, thanks for spotting and reporting this!
We recently had to move the website to a .php-free version and during the move I've overseen this typo.
We moved to using Hugo with a template from the Eclipse foundation, to updating the site is a two-step process - first update the content in the "website" repo and build it with Hugo (there is some description in the website/README) and then move only the generated static pages to "website-public" - this is the content used for the MAT website.

The second part should be automated, but it isn't (yet). Just make a PR for the "website" repo, and I'll take care about the rest.
If it is too confusing (I haven't got to the point to document the whole process after the move), just let me know and I would do the fixes.
